frontend/src/app/utils/address.tsx

```diff
@@ -19,3 +19,11 @@ export function formatAddress(address: Address) {
   }`;
   return result;
 }
+
```

Comment:
The `formatAddress` function adds `"null"` when `streetTwo` is `null`:

![null street](https://user-images.githubusercontent.com/49658917/112653550-ebbe0d80-8e24-11eb-838d-6b0bf5c3d69b.png)

Reply:
Good catch! Fixed `formatAddress` and added a test for this case. Also updated the snap shots that showed this